.github/workflows/release-build.yml

```diff
@@ -7,9 +7,9 @@ on:
     types: published
 
 jobs:
-################################
+#####################################
```

Comment:
Is there a reason all these are longer?

Reply:
Just to match the length of the longest header name